system-test/bigquery.ts

```diff
@@ -625,10 +625,15 @@ describe('BigQuery', () => {
     });
   });
 
-  describe('BigQuery/Model', () => {
+  describe.only('BigQuery/Model', () => {
     let model: Model;
+    const bucket = storage.bucket(generateName('bucket'));
+    const extractDest =
+      'gs://' + bucket.name + '/' + generateName('model-export');
+
+    before(async () => {
+      await bucket.create();
```

Comment:
I'm seeing a new bucket creation that happens every time, but no clean up.  Can we add an `after` hook here to remove the bucket we created?

Reply:
There's a `[deleteBuckets](https://github.com/googleapis/nodejs-bigquery/blob/01f9fe1c55cc92db63164084fbe01d0a95b510f8/system-test/bigquery.ts#L105)` method to clean up all the buckets created by the system tests